src/ansys/fluent/interface/post_objects/post_helper.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (C) 2022 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Provides a module for post objects."""
+
+import re
+
+from ansys.fluent.core.solver.flunits import get_si_unit_for_fluent_quantity
+from ansys.fluent.core.utils.fluent_version import FluentVersion
+
+
+class IncompleteISOSurfaceDefinition(RuntimeError):
+    """Raised when iso-surface definition is incomplete."""
+
+    def __init__(self):
+        """Initialize IncompleteISOSurfaceDefinition."""
+        super().__init__("Iso surface definition is incomplete.")
+
+
+class SurfaceCreationError(RuntimeError):
+    """Raised when surface creation is unsuccessful."""
+
+    def __init__(self):
+        """Initialize SurfaceCreationError."""
+        super().__init__("Surface creation is unsuccessful.")
+
+
+class PostAPIHelper:
+    """Class providing helper API for post objects."""
+
+    class _SurfaceAPI:
+        """Class providing APIs for surface operations."""
+
+        def __init__(self, obj):
+            self.obj = obj
+            self._surface_name_on_server = self.surface_name_on_server(obj._name)
+
+        @staticmethod
+        def surface_name_on_server(local_surface_name):
+            """Returns the surface name on server."""
+            return "_dummy_surface_for_pyfluent:" + local_surface_name.lower()
+
+        def _get_api_handle(self):
+            return self.obj.get_root().session.results.surfaces
+
+        def _delete_if_exist_on_server(self):
+            field_info = self.obj._api_helper.field_info()
+            surfaces_list = list(field_info.get_surfaces_info().keys())
+            if self._surface_name_on_server in surfaces_list:
+                self.delete_surface_on_server()
+
+        def create_surface_on_server(self):
+            """Creates the surface on server.
```

Comment:
Method docstring begins with a verb without an "s" or "es". 

Class descriptions, on the other hand, begin with verbs ending with "s" or "es".